packages/hash/frontend/src/blocks/userBlocks.tsx

```diff
@@ -0,0 +1,160 @@
+import { BlockMetadata } from "blockprotocol";
+import produce from "immer";
+import {
+  createContext,
+  Dispatch,
+  SetStateAction,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from "react";
+
+import { useCachedDefaultState } from "../components/hooks/useDefaultState";
+import { advancedFetch } from "../components/util/advancedFetch";
+
+export type UserBlock = BlockMetadata & {
```

Comment:
This blocks list is unique to each user, and is expanded upon by the specific remote blocks that the user decides to add. Based on your comment, I'm assuming the type should be named based on what it's holding rather than what it's used for. In that case, I think `RemoteBlockMetadata` is a more relevant name.